zapier/introduction/zapier-filtering/zapier-what-are-filters.md

```diff
@@ -0,0 +1,37 @@
+---
+author: kapnobatai136
+
+category: must-know
+
+aspects:
+  - introduction
+
+type: normal
+
+---
+
+# What Are Filters?
+
+---
+## Content
+
+Filters represent a very important part of Zapier because they allow you to control which data goes forward in the flow. Basically, if the received information does not fit a certain constraint (defined by you), then the flow won't continue. 
+
+Adding a filter is as easy as adding a new step, all you have to do is choose `'Filter by Zapier'` as your app.
```

Comment:
I don't think it's clear that a) filters are added as steps in the flow and b) to add one you can find them under an app called `Filter by Zapier`